.github/workflows/build.yml

```diff
@@ -362,3 +362,40 @@ jobs:
     - name: Execute headless tests
       working-directory: ${{ env.GITHUB_WORKSPACE }}
       run: python test.py -g --graphics=software
+
+  build_test_headless_alpine:
+    runs-on: ubuntu-latest
+    container: 
+      image: alpine:latest
+      options: --shm-size=8g
+    steps:
+    - name: Install Linux dependencies (Alpine)
+      run: |
+        apk add build-base wget git bash cmake python3 glu-dev sdl2-dev
+
+    - uses: actions/checkout@v4
+      with:
+        submodules: recursive
+
+    - name: Fix git detected dubious ownership in repository
+      run: |
+        chown -R $(id -u):$(id -g) $PWD
```

Comment:
Hmm.. interesting to know that dubious ownership can be solved this way too.

I was using this for dubious ownership issue on my workflow:
```
# Fix dubious ownership issue when running git describe inside a container
git config --global --add safe.directory "$PWD"
```